src/CqlBuilderPanel/codesSection/codesSubSection/codeSubSection/CodeSection.tsx

```diff
@@ -127,8 +128,9 @@ export default function CodeSection({
     ),
   };
 
-  function trimCodeInput() {
-    formik.setFieldValue("code", formik.values.code.trim());
+  function trimCodeInput(values) {
+    values.code = values.code.trim();
+    formik.setFieldValue("code", values.code);
```

Comment:
can we remove the line 132 and directly write 133 like: formik.setFieldValue("code", values.code.trim());

can you try if this working